doc/user/acl.md

```diff
@@ -0,0 +1,150 @@
+`TYPE:FLAGS:IDENTITY:PERMISSIONS`
+
+The contents of all the fields are case-sensitive.
+
+### Type
+
+The type of entry. Only one type of ACE is supported at this time.
+
+* A (Allow): Allow access to the specified identity for the given permissions.
+
+### Flags
+
+The flags provide additional information about how the ACE should be
+interpreted.
+
+* G (Group): The identity should be interpreted as a group.
+
+### Identity
+
+The identity (also called the principal) is specified in the name@domain format.
+The domain should be left off if the name is a UNIX user/group on the local
+domain. Currently, this is the only case supported by DAOS.
+
+There are three special identities, `OWNER@`, `GROUP@` and `EVERYONE@`,
+which align with User, Group, and Other from traditional POSIX permission bits.
+When providing them in the ACE string format, they must be spelled exactly as
+written here, in uppercase with no domain appended. The `GROUP@` entry must
+also have the `G` (group) flag.
+
+### Permissions
+
+The permissions in a resource's ACL permit a certain type of user access to
+the resource. Which types of permissions are valid depends on the resource
+type.
+
+| Permission	| Pool Meaning		| Container Meaning		|
+| ------------- | --------------------- | ----------------------------- |
+| r (Read)	| Alias for 't'		| Read data and attributes	|
+| w (Write)	| Alias for 'c' + 'd'	| Write data and attributes	|
+| c (Create)	| Create containers	| N/A				|
+| d (Delete)	| Delete any container	| Delete this container		|
+| t (Get-Prop)	| Connect/query		| Get container properties	|
+| T (Set-Prop)	| N/A			| Change container properties	|
+| a (Get-ACL)	| N/A			| Get container ACL		|
+| A (Set-ACL)	| N/A			| Change container ACL		|
+| o (Set-Owner)	| N/A			| Change owner user and group	|
+
+ACLs containing permissions not applicable to the given resource are considered
+invalid.
+
+In general a user/group's permissions must include at least some form of
+read access (for example, read or get-prop) before the user will be allowed to
+connect. A user with read-only *or* write-only permissions will be rejected when
+requesting RW access to a resource.
+
+### Denying Access
+
+Currently only "Allow" Access Control Entries are supported.
+
+However, it is possible to deny access to a specific user by creating an Allow
+entry for them with no permissions. This is fundamentally different from
+removing a user's ACE, which allows other ACEs in the ACL to determine their
+access.
+
+It is not possible to deny access to a specific group in this way, due to
+[the way group permissions are enforced](#enforcement).
+
+### ACE Examples
+
+* `A::daos_user@:rw`
+  * Allow the UNIX user named daos_user to have read-write access.
+* `A:G:project_users@:tc`
+  * Allow anyone in the UNIX group project_users to access a pool's contents and
+    create containers.
+* `A::OWNER@:rwdtTaAo`
+  * Allow the UNIX user who owns the container to have full control.
+* `A::EVERYONE@:r`
```

Comment:
Suggest adding one more example that uses the special principal GROUP@

Reply:
Done